addons/hr_holidays/hr_holidays.py

```diff
@@ -321,19 +321,32 @@ def onchange_date_to(self, cr, uid, ids, date_to, date_from):
 
         return result
 
+    def add_follower(self, cr, uid, ids, employee_id, context=None):
+        employee = self.pool['hr.employee'].browse(cr, uid, employee_id, context=context)
+        if employee and employee.user_id:
```

Comment:
we no longer need the `if foo and foo.bar_id` in 8.0, you are working on recordset so can directly use ``if foo.bar_id`